src/pages/user/order/OrderWait.tsx

```diff
@@ -0,0 +1,140 @@
+import styled from '@emotion/styled';
+import { colFlex, rowFlex } from '@styles/flexStyles';
+import OrderButton from '@components/user/order/OrderButton';
+import OrderStickyNavBar from '@components/user/order/OrderStickyNavBar';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import { useAtomValue } from 'jotai';
+import { adminOrdersAtom } from 'src/jotai/admin/atoms';
+import { userWorkspaceAtom } from 'src/jotai/user/atoms';
+import OrderAccountInfo from '@components/user/order/OrderAccountInfo';
+import HorizontalDivider from '@components/common/divider/HorizontalDivider';
+import { Color } from '@resources/colors';
+import { useEffect } from 'react';
+import useAdminOrder from '@hooks/admin/useAdminOrder';
+import useBlockPopState from '@hooks/useBlockPopState';
+
+const Container = styled.div`
+  width: 100%;
+  height: 100%;
+  padding-top: 50px;
+`;
+
+const SubContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 20px 20px 0 20px;
+  ${colFlex({ justify: 'center', align: 'center' })}
+`;
+
+const ContentsContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 12px 10px;
+  gap: 20px;
+  border-radius: 9px;
+  border: 0.5px solid #939393;
+  ${colFlex({ justify: 'center', align: 'start' })}
+`;
+
+const OrderInfoContainer = styled.div`
+  box-sizing: border-box;
+  width: 100%;
+  padding: 0 10px;
+  ${rowFlex({ justify: 'space-between', align: 'center' })};
+`;
+
+const StyledLabel = styled.div`
+  font-size: 13px;
+  font-weight: 600;
+`;
+
+const DescriptionContainer = styled.div`
+  width: 100%;
+`;
+
+const DescriptionText = styled.div`
+  box-sizing: border-box;
+  border-radius: 10px;
+  width: 100%;
+  font-size: 13px;
+  font-weight: 500;
+  color: #898989;
+  text-align: center;
+  box-sizing: border-box;
+  background: ${Color.LIGHT_GREY};
+  padding: 20px 30px;
+  word-break: keep-all;
+  ${colFlex({ justify: 'center', align: 'center' })}
+`;
+
+function OrderWait() {
+  const navigate = useNavigate();
+  const [searchParams] = useSearchParams();
+  const workspaceId = searchParams.get('workspaceId');
+  const tableNo = searchParams.get('tableNo');
+  const orderId = searchParams.get('orderId');
+  const isTossAvailable = searchParams.get('isTossAvailable') === 'true';
+
+  const workspace = useAtomValue(userWorkspaceAtom);
+  const adminOrders = useAtomValue(adminOrdersAtom);
+
+  const { fetchTodayOrders } = useAdminOrder(workspaceId || undefined);
```

Comment:
```suggestion
  const { fetchTodayOrders } = useAdminOrder(workspaceId);
```

기존에 이런식으로 undefined를 넣는 방식을 사용했었나요??

Reply:
음 그렇긴 한데요 지금 보니깐 애초에 useAdminOrder를 사용하면 안되는 페이지네요.
orderId로만 fetch하게끔 해주세요~

Reply:
아뇨 realTimeOrder를 사용하면 안되죠..?

여기는 admin이 아닌 user가 보는 화면이고, 우리가 가지고 있는 정보는 workspaceId, tableNo, orderId만 있는 상황입니다.

현재 orderId를 통해서 주문을 얻어올 수 있으니 interval을 사용해서 order를 fetch하시라는 말이었습니다.

Reply:
아뇨 해당 메소드 보시면 orderId를 파라매터로 받는거 보실 수 있습니다.